fractal/parser.py

```diff
@@ -511,9 +525,18 @@
     type=int,
     help="Workflow ID",
 )
-workflow_add_task_parser.add_argument(
-    "task_id_or_name", help="ID or name of the new task", type=str
+
+workflow_add_task_id_or_name_group = (
+    workflow_add_task_parser.add_mutually_exclusive_group(required=True)
+)
+workflow_add_task_id_or_name_group.add_argument(
+    "--task-id", help="ID of the task to edit", type=int
 )
+workflow_add_task_id_or_name_group.add_argument(
+    "--task-name", help="Name of the task to edit", type=str
+)
+
+
```

Comment:
I guess that the most consistent choice here is to rename `--version` into `--task-version`

Reply:
Also: `"--version", type=str, help="Version of the new task"` should be made consistent with help messages for `--task-name` and `--task-id`